src/NLog/Internal/Fakeables/IAppEnvironment.cs

```diff
@@ -33,14 +33,28 @@
 
 namespace NLog.Internal.Fakeables
 {
+    using System.Collections.Generic;
+    using System.Xml;
+
     /// <summary>
     /// Abstract calls to File
     /// </summary>
-    internal interface IFile
+    internal interface IAppEnvironment
```

Comment:
Why is this interface so fat? It's hard to reuse and doesn't follow the SRP. :/

Reply:
Like you said it is an internal interface and it is the properties needed to mock the loading of NLog-config.

Reply:
Please split it into separate interfaces. It's also important for internal code for a proper design

Reply:
Have split into IFileSystem and IAppEnvironment

Reply:
Thanks!